docs/source/tutorials/e2e_flow.rst

```diff
@@ -348,101 +411,143 @@ conversion is that you can directly work with standard formats. This helps
 with interoperability with other libraries since torchtune doesn't add yet
 another format to the mix.
 
-Let's take a look at an example of how this would work with a popular codebase
-used for running performant inference with LLMs -
-`gpt-fast <https://github.com/pytorch-labs/gpt-fast/tree/main>`_. This section
-assumes that you've cloned that repository on your machine.
+Let's start with huggingface
 
-``gpt-fast`` makes some assumptions about the checkpoint and the availability of
-the key-to-file mapping i.e. a file mapping parameter names to the files containing them.
-Let's satisfy these assumptions, by creating this mapping
-file. Let's assume we'll be using ``<new_dir>/Llama-2-7B-hf`` as the directory
-for this. ``gpt-fast`` assumes that the directory with checkpoints has the
-same format at the HF repo-id.
+**Case 1: HF using BASE MODEL + trained adapter**
```

Comment:
The three different cases are not super clear to me. Might be best to explain them in a sentence or two in advance so as to not be too confusing (e.g. when should I use which one). Also why is "base model" capitalized?

Reply:
Trying to differentiate vs merged model. I will make it clear